src/storage/realm/schema/vault.ts

```diff
@@ -271,6 +271,7 @@ export const VaultSchema: ObjectSchema = {
     specs: RealmSchema.VaultSpecs,
     archived: 'bool',
     archivedId: 'string?',
+    isMigrating: 'bool',
```

Comment:
Shouldn't isMigrating attribute be optional? As for the already existing vaults, it would be non-existent.

Reply:
The migration version update defaults it to false